scaaml/capture/input_generators/attack_point_iterator_models.py

```diff
@@ -165,3 +166,177 @@ def items(self) -> AttackPointIteratorT:
     def get_generated_keys(self) -> List[str]:
         """Returns an exhaustive list of names this iterator will create."""
         return self.configuration.get_generated_keys()
+
+
+class ZipIteratorModel(BaseModel):
+    """
+    Attack point iterator zip pydantic model. This class takes any amount of
+    operands and combines them just like the `zip` function in Python.
```

Comment:
This docstring is a little misleading. Because `zip` iterates over tuples. However this iterates over dictionaries.